webserver/webserver/src/main/java/io/helidon/webserver/http1/Http1ServerResponse.java

```diff
@@ -411,7 +417,8 @@ private BlockingOutputStream(ServerResponseHeaders headers,
             this.contentLength = headers.contentLength().orElse(-1);
             this.request = request;
             this.keepAlive = keepAlive;
-            this.forcedChunked = headers.contains(HeaderValues.TRANSFER_ENCODING_CHUNKED);
+            this.forcedChunked = headers.contains(HeaderValues.TRANSFER_ENCODING_CHUNKED)
```

Comment:
why do we force chunked encoding on trailers? There should be no reason for this